pyaedt/modeler/cad/Primitives3D.py

```diff
@@ -153,6 +153,8 @@ def create_cylinder(self, cs_axis, position, radius, height, numSides=0, name=No
         if isinstance(radius, (int, float)) and radius < 0:
             raise ValueError("Radius must be greater than 0.")
 
+        assert len(position) == 3, "Position Argument must be a valid 3 Element List"
```

Comment:
The message string should be a complete sentence using sentence case and punctuation and follow style guidelines: Position argument must be a valid three-element list. If you can somehow set off (tag) "Position" that would be fine.